services/movie_session.py

```diff
@@ -25,10 +25,10 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
 
 
 def update_movie_session(
-    session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None,
```

Comment:
if argument can be None, you should show in type hint
```suggestion
        show_time: str | None = None,
        movie_id: int | None = None,
        cinema_hall_id: int | None = None,
```